ProtoWorkspace/BufferTracker.fs

```diff
@@ -6,29 +6,25 @@ open System.Collections.Generic
 open Microsoft.FSharp.Compiler
 open Microsoft.FSharp.Compiler.Range
 
-
 [<NoComparison>]
-type EditorBuffer =
-    {   Text           : string
-        Range          : range 
-        IsDirty        : bool
-        Encoding       : Encoding
-        LastChangeTime : DateTime
-        ViewCount      : int 
-    }
+type EditorBuffer = 
+    { Text : string
+      Range : range
+      IsDirty : bool
+      Encoding : Encoding
+      LastChangeTime : DateTime
+      ViewCount : int }
```

Comment:
I really dislike this auto formatting rule
```fsharp
type EditorBuffer = {   
    Text : string
    Range : range
    IsDirty : bool
    Encoding : Encoding
    LastChangeTime : DateTime
    ViewCount : int
}
```
is clearly the best way to format a record 😝